app/services/module_time_to_complete.rb

```diff
@@ -11,9 +11,9 @@ def initialize(user:, training_module_id:)
 
   # @return [Hash{String => Integer}]
   def update_time(training_module_id)
-    # binding.pry
     @training_module_id = training_module_id
```

Comment:
Question: Does it make sense for this class to instead be initialised without any module named? It can loop over all published modules if it wants. Does it then need a method to target a specific module? Update one vs update all...